components/fxa-client/src/ffi_types.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto2";
+
+package ffi_types;
+
+message Profile {
+  optional string uid = 1;
```

Comment:
I marked them as optional because in theory the FxA servers mark these fields as all optional (we should probably correct the Rust code here to reflect that). In reality we always get `uid` and friends.